src/fides/cli/__init__.py

```diff
@@ -89,7 +99,11 @@ def cli(ctx: click.Context, config_path: str, local: bool) -> None:
 
     # Check the server health and version if an API command is invoked
     if ctx.invoked_subcommand in SERVER_CHECK_COMMAND_NAMES:
-        check_server(VERSION, str(config.cli.server_url), quiet=True)
+        check_server(fides_version, str(config.cli.server_url), quiet=True)
+
+    # Enforce that the fidesctl-plus API is available for plus-only commands
+    if ctx.invoked_subcommand in PLUS_COMMANDS:
+        check_plus_server(str(config.cli.server_url), quiet=True)
```

Comment:
I'm not sure how to test this without creating a new `ethyca-fides` release and importing it into fidesctl-plus. Doing so would then require transitioning fidesctl-plus from `fidesctl` --> `ethyca-fides`, which is a whole separate project.

Reply:
yeah, we knew this day would come 🙁